server/konnection/urls.py

```diff
@@ -16,9 +16,11 @@
 from django.contrib import admin
 from django.urls import path
 from django.urls.conf import include
+from posts import views as posts_views
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('service/author/', include('posts.urls')),
     path('service/author/', include('author.urls')),
+    path('service/public/', posts_views.PublicPostView.as_view(), name='public'),
```

Comment:
it would be good to have not call the view directly from the main urls. Suggestion, move this public url into post url

Reply:
Thanks for catching that! Moved '/author' and '/public' into posts url.

Reply:
`path('service/public/', posts_views.PublicPostView.as_view(), name='public')`
this can now be included in your post/url